bitnami/kubernetes-event-exporter/Chart.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: kubernetes-event-exporter
+description: This tool allows exporting the often missed Kubernetes events to various outputs so that they can be used for observability or alerting purposes.
+version: 1.0.0
+appVersion: 0.9
```

```suggestion
appVersion: 0.9.0
```
Just a minor change, as our internal tools expects `appVersion` to follow semver standards.

---

I agree that everyone should use SemVer for versioning their stuff. But the upstream version of this event exporter only uses ‘0.9‘. IMHO we should use it like this. But when your tests are failing because of this, I agree for going the easy way :-)

---

Yes, this was something that also bothered me. I looked into it and the error comes from our tool expecting a string but receiving a float. As such, I think that this can be solved by just changing it to `appVersion: v0.9`.